index.html

```diff
@@ -744,6 +745,11 @@ <h3>Validate Algorithm</h3>
 <a href="#bitstring-expansion-algorithm">Bitstring Expansion Algorithm</a>.
         </li>
         <li>
+If the length of the |revocation bitstring| divided by
+<a href="#size">`size`</a> is less than 131,072, raise a
```

Comment:
```suggestion
<a href="#statusSize">`statusSize`</a> is less than 131,072, raise a
```

Reply:
I'm going to hold off on this change until the change happens in #109.

Reply:
This change is now applied in #109.